components/Hyauctions/Hyauctions.tsx

```diff
@@ -0,0 +1,18 @@
+import Link from 'next/link'
+import React from 'react'
+
+function Hyauctions() {
+    return (
+        <>
+            <h2>Coflnet</h2>
+            <hr />
+            <p>
+                We provide a hyauctions alternative. That includes player and item history but also bazaar history, notifications,{' '}
+                <Link href={'/crafts'}>crafts</Link> & <Link href={'/kat'}>kat flips</Link> as well as an <Link href={'/flipper'}>advanced ah flipper</Link>.
+            </p>
+            <p>Just try using the search bar above (it actually works).</p>
```

Comment:
@Ekwav The "it actually works" sounds kinda weird. Why do we need to emphasize that a feature works?

Reply:
because hyauctions is well know  for that their search doesn't work/ doesn't find things